src/web/mjs/engine/Storage.mjs

```diff
@@ -911,4 +912,18 @@ export default class Storage {
             } );
         } );
     }
+
+    get downloadedConnectors() {
```

Comment:
Just return a list of connector IDs (string), makes further processing much simpler.